src/HIE/Bios/Cradle.hs

```diff
@@ -455,22 +457,32 @@ callableToProcess (Program path) file = do
   return $ proc canon_path (maybeToList file)
 
 ------------------------------------------------------------------------
+
+projectFileProcessArgs :: CradleProjectConfig -> [String]
+projectFileProcessArgs (ExplicitConfig prjFile) = ["--project-file", prjFile]
+projectFileProcessArgs NoExplicitConfig = []
+
+projectLocationOrDefault :: CradleProjectConfig -> [FilePath]
+projectLocationOrDefault = \case
+  NoExplicitConfig -> ["cabal.project", "cabal.project.local"]
+  (ExplicitConfig prjFile) -> [prjFile, prjFile <.> "local"]
```

Comment:
Should the explicit config include the .local file? Or will cabal pick it up automatically if it has the main project file?

Reply:
This is used for cabal cradle dependencies, e.g. telling HLS that the cradle needs to be reloaded if either of the two changes. IIRC, that behaviour is in line with cabal.
However, the name is misleading.